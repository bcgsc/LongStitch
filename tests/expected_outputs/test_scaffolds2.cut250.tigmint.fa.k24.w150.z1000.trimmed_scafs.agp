ntLink_0	1	6717	1	W	189612::189612:0-6717	1	6717	-
ntLink_0	6718	6831	2	N	114	scaffold	yes	paired-ends
ntLink_0	6832	19685	3	W	190014::190014:0-13076	1	12854	+
ntLink_0	19686	21370	4	W	188379::188379:0-1767	1	1685	-
ntLink_0	21371	21390	5	N	20	scaffold	yes	paired-ends
ntLink_0	21391	25203	6	W	188339::188339:0-4163	351	4163	-
ntLink_0	25204	29069	7	W	190319::190319:0-4194	329	4194	+
ntLink_0	29070	29221	8	N	152	scaffold	yes	paired-ends
ntLink_0	29222	39234	9	W	188338::188338:0-10013	1	10013	+
ntLink_0	39235	39464	10	N	230	scaffold	yes	paired-ends
ntLink_0	39465	45045	11	W	182984::182984:0-5581	1	5581	-
ntLink_0	45046	45592	12	N	547	scaffold	yes	paired-ends
ntLink_0	45593	163706	13	W	190166::190166:0-118114	1	118114	-
ntLink_1	1	2346	1	W	182922::182922:0-2346	1	2346	-
ntLink_1	2347	2456	2	N	110	scaffold	yes	paired-ends
ntLink_1	2457	4863	3	W	9127::9127:0-2407	1	2407	-
ntLink_1	4864	4891	4	N	28	scaffold	yes	paired-ends
ntLink_1	4892	22778	5	W	190488::190488:0-17887	1	17887	-
ntLink_1	22779	23590	6	N	812	scaffold	yes	paired-ends
ntLink_1	23591	48604	7	W	183282::183282:0-25014	1	25014	+
ntLink_1	48605	48940	8	N	336	scaffold	yes	paired-ends
ntLink_1	48941	57895	9	W	189475::189475:0-8955	1	8955	-
40595::40595:0-1190	1	1190	1	W	40595::40595:0-1190	1	1190	+
190963::190963:0-27311	1	27311	1	W	190963::190963:0-27311	1	27311	+
