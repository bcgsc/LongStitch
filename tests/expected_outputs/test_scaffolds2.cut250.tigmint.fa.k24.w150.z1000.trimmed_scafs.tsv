189475::189475:0-8955	0	8955
188379::188379:0-1767	0	1685
9127::9127:0-2407	0	2407
189612::189612:0-6717	0	6717
40595::40595:0-1190	0	1190
190014::190014:0-13076	0	12854
182922::182922:0-2346	0	2346
190166::190166:0-118114	0	118114
182984::182984:0-5581	0	5581
190319::190319:0-4194	328	4194
183282::183282:0-25014	0	25014
190488::190488:0-17887	0	17887
188338::188338:0-10013	0	10013
190963::190963:0-27311	0	27311
188339::188339:0-4163	350	4163
